src/js/serial_cordova.js

```diff
@@ -0,0 +1,576 @@
+'use strict';
```

Comment:
This is mostly duplicated lines from `serial.js`, which will be a maintenance and testing nightmare, and broken more often than not - the code should be refactored into a class for the common functionality, and two classes for only the per-architecture functionality. See also https://sonarcloud.io/component_measures?id=betaflight_betaflight-configurator&metric=new_duplicated_lines_density&pullRequest=1946&selected=betaflight_betaflight-configurator%3Asrc%2Fjs%2Fserial_cordova.js&view=list

Reply:
Is the `cordova_chromeapi.js` a better approach for maintenance and testing ? I try to emulate chrome APIs with available cordova plugins.
With this solution, `serial.js` doesn't change.